.github/workflows/publish.yml

```
@@ -9,7 +9,7 @@ jobs:
   version-check:
     # We need this job to run only on push with tag.
     if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Please, fix the version to avoid problems after the ubuntu version update in the future.
```suggestion
    runs-on: ubuntu-24.04
```